src/index.ts

```diff
@@ -0,0 +1,18 @@
+import express, { Response, Request } from 'express'
+import cors from 'cors'
+import databaseService from './database/database.services'
+
+const app = express()
+const port = 3000
+app.use(express.json())
+app.use(cors())
+
+databaseService.connect()
+
+app.get('/', cors(), (req: Request, res: Response) => {
+    res.send('Hello Developer')
+})
+
```

Comment:
this use not good, check more. detail here 
https://viblo.asia/p/cors-la-gi-cors-voi-nodejs-Qbq5QyyL5D8